Services/WebDriver/Client.cpp

```diff
@@ -50,12 +50,40 @@ ErrorOr<NonnullRefPtr<Session>, Web::WebDriver::Error> Client::find_session_with
     return Web::WebDriver::Error::from_code(Web::WebDriver::ErrorCode::InvalidSessionId, "Invalid session id");
 }
 
+// https://w3c.github.io/webdriver/#dfn-close-the-session
 void Client::close_session(String const& session_id)
 {
+    // FIXME: 1. If session's HTTP flag is set, remove session from active HTTP sessions.
+
+    // 2. Remove session from active sessions.
     if (s_sessions.remove(session_id))
         dbgln_if(WEBDRIVER_DEBUG, "Shut down session {}", session_id);
     else
         dbgln_if(WEBDRIVER_DEBUG, "Unable to shut down session {}: Not found", session_id);
+
+    // 3. Perform the following substeps based on the remote end's type:
+    // 4. Remote end is an endpoint node
```

Comment:
The numbering here is wrong, this is a `->` part of step 3.

Reply:
woops, fixed